aten/src/ATen/Dispatch.h

```diff
@@ -416,6 +416,46 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                       \
   }()
 
+#define AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES_AND3(                        \
```

Comment:
This will tentatively conflict with https://github.com/pytorch/pytorch/pull/77479/files depending on which is merged fist.

Reply:
Don't remove till that PR is merged. (It is required if this is merged first!)